docs/wcs/examples/from_file.py

```diff
@@ -40,6 +41,22 @@ def load_wcs_from_file(filename):
     # some floating-point error.
     assert np.max(np.abs(pixcrd - pixcrd2)) < 1e-6
 
+    # As illustrated below, origin defines whether the origin pixel position
+    # is 0- or 1-indexed.
+    x_origin = 0
+    y_origin = 0
+    origin = 0
+    assert (w.wcs_pix2world(x_origin, y_origin, origin) ==
+            w.wcs_pix2world(x_origin + 1, y_origin + 1, origin + 1))
+
+    # Following the origin logic from above, X=0 and Y=0 are actually
+    # not within the image when origin=1.
+    # This example passes in X=0 and Y=0 for origin=1, and converts the result
+    # back to pixels for origin=0. The final result of (-1, -1) shows that
+    # pixels are out-of-bounds.
```

Comment:
Perhaps `outside the image` is better than `out-of-bounds`.

Reply:
I deleted this block because I find that it is no longer necessary with the other changes you suggested above.